ingest/response.go

```diff
@@ -0,0 +1,38 @@
+package ingest
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	uuid "github.com/satori/go.uuid"
+)
+
+type Response struct {
+	Success   bool                     `json:"success"`
+	Message   string                   `json:"message"`
+	Errors    []map[string]interface{} `json:"errors"`
+	RequestId uuid.UUID
+}
+
+func convertHttpToIngestResponse(resp *http.Response) (*Response, error) {
+	ingestResponse := &Response{}
+
+	body, err := ioutil.ReadAll(resp.Body)
```

Comment:
you can use a json decoder to parse the response

Reply:
What do you mean? I am converting **io.ReadCloser** to **[]byte** here and parsing json below.